pkg/policy/distillery.go

```diff
@@ -153,6 +153,45 @@ func (cache *PolicyCache) UpdatePolicy(identity *identityPkg.Identity) error {
 	return err
 }
 
+// GetAuthTypes returns the AuthTypes required by the policy between the localID and remoteID, if
+// any, otherwise returns nil.
```

Comment:
nit: `ID` can end up ambiguous, it'd be nice to give the reader one less thing to be confused about - stating clearly that the inputs are expected to be security identities. If by any weird course we end up plugging the wrong id numbers in, it could help point out the mistake.
```suggestion
// GetAuthTypes returns the AuthTypes required by the policy between the
// security identities localID and remoteID, if any, otherwise returns nil.
```